### ISSUE TITLE
Can't take Screenshots on Android 6 &7

### ISSUE THREAD

**Issue by KrasserTommy** [nisrulz/screenshott#6]
Please fill out the blanks below describing your issue

What are the steps to reproduce this issue?
-------------------------------------------
1. …
2. …
3. …
…

What happens?
-------------
In the Android Studio Emulator and on my own device the Screenshot is made on Android 5 (Lollipo). On other devices (Android 6 & 7) I can't find the Screenshot. Do you know where I can find it?

What were you expecting to happen?
----------------------------------
…

Any logs, error output, bugreport etc?
----------------------------
No errors.

Any other comments?
-------------------
…

What versions of software are you using?
----------------------------------------
**Device Information:** …

**Android Version:** …

**Configuration Information:** …

**Comment by nisrulz** [comment]
@KrasserTommy Let me cross-check. Thank you for reporting this.

**Comment by nisrulz** [comment]
@KrasserTommy Are you talking about the filepath? If so then have a look at issue #5

**Comment by KrasserTommy** [comment]
I don't think so. On my Lollipop Device the Picture is saved to the Pictures Folder. On my Android 6 and 7 Device the folder is empty...

**Comment by nisrulz** [comment]
I am guessing you haven't provided the write permissions. Can you check your logcat output, where it would show if that is the case.

**Comment by nisrulz** [comment]
@KrasserTommy Any updates ? I was able to take screenshots in both Android 6 and 7.
